mono/tests/Makefile.am

```diff
@@ -1414,6 +1414,14 @@ test-console-output: console-output.exe
 	@diff -w console-output.exe.stdout $(srcdir)/console-output.exe.stdout.expected \
 		&& diff -w console-output.exe.stderr $(srcdir)/console-output.exe.stderr.expected
 
+PROCESS_STRESS_TESTS=	\
+		process-stress-1.exe	\
+		process-stress-2.exe	\
+		process-stress-3.exe
+
+test-process-stress: $(PROCESS_STRESS_TESTS) test-runner.exe
```

Comment:
They are more intended to run on demand when working on the Process implementation to check an array of common race conditions.